docs/api/apiv3/paths/days_non_working.yml

```diff
@@ -1,5 +1,51 @@
 # /api/v3/days/non_working
 ---
+get:
+  summary: Lists all non working days
+  operationId: list_non_working_days
+  tags:
+    - Work Schedule
+  description: |-
+    Lists all one-time non working days, such as holidays.
+    It does not lists the non working weekdays, such as each Saturday, Sunday.
+    For listing the weekends, the `/api/v3/days` endpoint should be used.
+
+    All days from current year are returned by default.
+  parameters:
+    - name: filters
+      in: query
+      description: |-
+        JSON specifying filter conditions.
+
+        Accepts the same format as returned by the [queries](https://www.openproject.org/docs/api/endpoints/queries/)
+        endpoint. Currently supported filters are:
+
+        + date: the inclusive date interval to scope days to look up. When
+          unspecified, default is from the beginning of current month to the end
+          of following month.
```

Comment:
Isn't it from beginning of current year to end of current year?

Reply:
Nope, it is correct.